frontend/openchat-shared/src/domain/chat/chat.ts

```diff
@@ -661,10 +667,9 @@ export type LocalMessageUpdates = {
 
 export type EventsResponse<T extends ChatEvent> = "events_failed" | EventsSuccessResult<T>;
```

Comment:
Do `EventsResponse` and `EventsSuccessResult` even _need_ to be generic any more?